web2/api/get-my-counteroffers.gql

```diff
@@ -0,0 +1,51 @@
+query GetMyCounterOffers($first: Int, $skip: Int, $owner: Bytes!) {
```

Comment:
When we will be refactoring this, we need to introduce [GraphQL Fragments](https://www.apollographql.com/docs/react/data/fragments/) for shared logic